diff --git a/dist/trezor-keyring.cjs b/dist/trezor-keyring.cjs
index c8eb4a45ebf19b4c365e49b5de03b04f0b1b48aa..eb5c9a668bd86df6225cc5121a5da63355be55bd 100644
--- a/dist/trezor-keyring.cjs
+++ b/dist/trezor-keyring.cjs
@@ -19,10 +19,12 @@ const hdkey_1 = __importDefault(require("hdkey"));
 const hdPathString = `m/44'/60'/0'/0`;
 const SLIP0044TestnetPath = `m/44'/1'/0'/0`;
 const legacyMewPath = `m/44'/60'/0'`;
+const bip45Path = `m/45/60/0`;
 const ALLOWED_HD_PATHS = {
     [hdPathString]: true,
     [legacyMewPath]: true,
     [SLIP0044TestnetPath]: true,
+    [bip45Path]: true,
 };
 const keyringType = 'Trezor Hardware';
 const pathBase = 'm';
